controllers/storageclassclaim/storageclassclaim_controller.go

```diff
@@ -196,6 +197,12 @@ func (r *StorageClassClaimReconciler) reconcileConsumerPhases() (reconcile.Resul
 			},
 		}
 		if err = r.get(existing); err == nil {
+			storageClassNamespace := existing.Annotations[storageClassNamespaceAnnotation]
```

Comment:
@nb-ohad can this cause problem for the upgrades as the annotation is not set in the SC?